fastdds_python/test/types/CMakeLists.txt

```diff
@@ -41,6 +41,20 @@ find_package(fastdds 3 REQUIRED)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+if(NOT WIN32)
```

Comment:
This CMakeLists are generated by `fastddsgen`. We need a PR to fastddsgen adding the code.